hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3ExpiredMultipartUploadsAbortRequest.java

```diff
@@ -259,6 +289,8 @@ private void updateTableCache(OzoneManager ozoneManager,
 
           String multipartOpenKey;
           try {
+            KeyArgs keyArgs = buildKeyArgs(multipartUpload);
+            addOrGetMissingDirectories(ozoneManager, keyArgs, trxnLogIndex);
```

Comment:
This should have a corresponding OM response to add the missing directories to the DB batch similar to https://github.com/apache/ozone/pull/6496?

Reply:
Let me know your thoughts on my reply related to https://github.com/apache/ozone/pull/6496